worker/create-alerts.ts

```diff
@@ -0,0 +1,61 @@
+import {
+    CloudWatchClient,
+    DeleteAlarmsCommand,
+    DescribeAlarmsCommand,
+    PutMetricAlarmCommand,
+} from '@aws-sdk/client-cloudwatch';
+import { env } from '../app/env';
+import cronRunMetric from '../modules/metrics/cron.metric';
+import { WorkerJob } from './manual-jobs';
+
+export async function createAlertsIfNotExist(jobs: WorkerJob[]): Promise<void> {
+    const cloudWatchClient = new CloudWatchClient({
+        region: process.env.AWS_REGION,
```

Comment:
This is an assumption that process.env.AWS_REGION will always be present by default. Correct?

Reply:
it has to be manually put into the environment variables, its not injected by AWS or something if you mean this